billing/payment-method.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Add or update a payment method
+description: Information on how to add or update a payment method in Docker Hub
+keywords: payments, billing, subscription
+---
+
+Find information on how to add or update edit a payment method for your personal account or for an organization. 
+
+You can add a payment method or update your account's existing payment method at any time.
```

Comment:
**chore:** I think it is worth mentioning that the last payment method can't be removed unless the subscription is downgraded to a free plan.